configs/platforms/fedora-f25-i386.rb

```diff
@@ -0,0 +1,10 @@
+platform "fedora-f25-i386" do |plat|
+  plat.servicedir "/usr/lib/systemd/system"
+  plat.defaultdir "/etc/sysconfig"
+  plat.servicetype "systemd"
+
+  plat.provision_with "/usr/bin/dnf install -y --best --allowerasing autoconf automake rsync gcc make rpmdevtools rpm-libs"
+  plat.add_build_repository "http://pl-build-tools.delivery.puppetlabs.net/yum/pl-build-tools-release-#{plat.get_os_name}-25.noarch.rpm"
```

Comment:
The pattern used through most of the RPM based platforms is to use 

`pl-build-tools-release-#{plat.get_os_name}-#{plat.get_os_version}.noarch.rpm`

For fedora, however, get_os_version returns "f24" instead of just "24", so we don't use that for the fedora platforms, just get_os_name. It's presumably a way to make it easier to copy platform files from version to version.